### PR TITLE
[CloudService] Second preview release for CloudService

### PR DESCRIPTION
**Pull request by wyunchi-ms**
## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by BethanyZhou** [comment]
Hi @wyunchi-ms , please add Az.CloudService to Az.psd1

File: src/CloudService/Changelog.md [hunk 1]
```diff
@@ -19,6 +19,9 @@
 -->
 ## Upcoming Release
 
+## Version 1.0.0
+* GA release for module Az.CloudService.
```
Comment (dingmeng-xue) [inline src/CloudService/Changelog.md:23]: ## Version 0.2.0
* Bump API version to stable 2021-03-01
* Supported OSFamily and OSVersion

File: src/CloudService/Az.CloudService.psd1 [hunk 2]
```diff
@@ -127,7 +127,7 @@ PrivateData = @{
         # IconUri = ''
 
         # ReleaseNotes of this module
-        # ReleaseNotes = ''
+        ReleaseNotes = '* GA release for module Az.CloudService.'
```
Comment (dingmeng-xue) [inline src/CloudService/Az.CloudService.psd1:130]: * Bump API version to stable 2021-03-01
* Supported OSFamily and OSVersion

File: src/CloudService/Az.CloudService.psd1 [hunk 3]
```diff
@@ -12,7 +12,7 @@
 RootModule = './Az.CloudService.psm1'
 
 # Version number of this module.
-ModuleVersion = '0.1.0'
+ModuleVersion = '1.0.0'
```
Comment (dingmeng-xue) [inline src/CloudService/Az.CloudService.psd1:15]: 0.2.0

File: src/CloudService/Properties/AssemblyInfo.cs [hunk 4]
```diff
@@ -24,5 +24,5 @@
 [assembly: ComVisible(false)]
 [assembly: CLSCompliant(false)]
 [assembly: Guid("1fe4b4c7-44d2-4800-a935-733381e663ae")]
-[assembly: AssemblyVersion("0.1.0")]
-[assembly: AssemblyFileVersion("0.1.0")]
+[assembly: AssemblyVersion("1.0.0")]
```
Comment (dingmeng-xue) [inline src/CloudService/Properties/AssemblyInfo.cs:27]: 0.2.0

File: src/CloudService/Properties/AssemblyInfo.cs [hunk 5]
```diff
@@ -24,5 +24,5 @@
...
+[assembly: AssemblyFileVersion("1.0.0")]
```
Comment (dingmeng-xue) [inline src/CloudService/Properties/AssemblyInfo.cs:28]: 0.2.0

File: tools/VersionController/MinimalVersion.csv [hunk 6]
```diff
@@ -1,2 +1,3 @@
 "Module","Minimal Version"
-"Az.RedisEnterpriseCache","1.0.0"
\ No newline at end of file
+"Az.RedisEnterpriseCache","1.0.0"
+"Az.CloudService","1.0.0"
```
Comment (dingmeng-xue) [inline tools/VersionController/MinimalVersion.csv:3]: Should be roll back.

**Review by dingmeng-xue — APPROVED** [review]

**Pull request merged**